src/dot_dotfiles/zsh/BUILD.bazel

```diff
@@ -0,0 +1,35 @@
+load("@bazel_skylib//rules:diff_test.bzl", "diff_test")
+load("//tools/chezmoi:rules.bzl", "chezmoi_execute_template")
+
+package(default_visibility = ["//visibility:public"])
+
+chezmoi_execute_template(
+    name = "config_zsh",
+    src = "config.zsh.tmpl",
+    out = "config.zsh",
+    data_file = "test_data.toml",
```

Comment:
Can we instead use the actual data file in the repo exposed as a dependency (like a file group or singular file or something?